packages/config-utils/proxy.js

```diff
@@ -55,7 +55,10 @@ module.exports = ({
         // stage-stable / stage-beta branches don't exist in build repos
         // Currently stage pulls from QA
         env = env.replace('stage', 'qa');
-        // QA and stage are deployed with Akamai which requires a corporate proxy
+    }
+
+    if (env.startsWith('prod') || env.startsWith('qa')) {
```

Comment:
Shouldn't here be also `stage` condition?

Reply:
You know what, lets make it a bit cleaner :wink: